<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>UseArrowWidgetWindow</class>
 <widget class="QWidget" name="UseArrowWidgetWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>945</width>
    <height>667</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <layout class="QGridLayout" name="gridLayout">
   <item row="0" column="0">
    <widget class="QSplitter" name="splitter">
     <property name="orientation">
      <enum>Qt::Horizontal</enum>
     </property>
     <widget class="QGroupBox" name="groupBox">
      <property name="maximumSize">
       <size>
        <width>300</width>
        <height>16777215</height>
       </size>
      </property>
      <property name="title">
       <string>Config</string>
      </property>
      <layout class="QGridLayout" name="gridLayout_4">
       <item row="0" column="0">
        <layout class="QGridLayout" name="gridLayout_3">
         <item row="1" column="0">
          <widget class="QGroupBox" name="groupBox_4">
           <property name="title">
            <string>Point2</string>
           </property>
           <layout class="QGridLayout" name="gridLayout_6">
            <item row="0" column="0">
             <layout class="QHBoxLayout" name="horizontalLayout_2">
              <item>
               <widget class="QLabel" name="label">
                <property name="text">
                 <string>X</string>
                </property>
               </widget>
              </item>
              <item>
               <widget class="QDoubleSpinBox" name="pot2xSpinBox">
                <property name="singleStep">
                 <double>0.010000000000000</double>
                </property>
               </widget>
              </item>
              <item>
               <widget class="QLabel" name="label_2">
                <property name="text">
                 <string>Y</string>
                </property>
               </widget>
              </item>
              <item>
               <widget class="QDoubleSpinBox" name="pot2ySpinBox">
                <property name="singleStep">
                 <double>0.010000000000000</double>
                </property>
               </widget>
              </item>
              <item>
               <widget class="QLabel" name="label_3">
                <property name="text">
                 <string>Z</string>
                </property>
               </widget>
              </item>
              <item>
               <widget class="QDoubleSpinBox" name="pot2zSpinBox">
                <property name="singleStep">
                 <double>0.010000000000000</double>
                </property>
               </widget>
              </item>
              <item>
               <spacer name="horizontalSpacer">
                <property name="orientation">
                 <enum>Qt::Horizontal</enum>
                </property>
                <property name="sizeHint" stdset="0">
                 <size>
                  <width>40</width>
                  <height>20</height>
                 </size>
                </property>
               </spacer>
              </item>
             </layout>
            </item>
           </layout>
          </widget>
         </item>
         <item row="0" column="0">
          <widget class="QGroupBox" name="groupBox_3">
           <property name="title">
            <string>Point1</string>
           </property>
           <layout class="QGridLayout" name="gridLayout_5">
            <item row="1" column="0">
             <layout class="QHBoxLayout" name="horizontalLayout">
              <item>
               <widget class="QLabel" name="label_4">
                <property name="text">
                 <string>X</string>
                </property>
               </widget>
              </item>
              <item>
               <widget class="QDoubleSpinBox" name="pot1xSpinBox">
                <property name="singleStep">
                 <double>0.010000000000000</double>
                </property>
               </widget>
              </item>
              <item>
               <widget class="QLabel" name="label_5">
                <property name="text">
                 <string>Y</string>
                </property>
               </widget>
              </item>
              <item>
               <widget class="QDoubleSpinBox" name="pot1ySpinBox">
                <property name="singleStep">
                 <double>0.010000000000000</double>
                </property>
               </widget>
              </item>
              <item>
               <widget class="QLabel" name="label_6">
                <property name="text">
                 <string>Z</string>
                </property>
               </widget>
              </item>
              <item>
               <widget class="QDoubleSpinBox" name="pot1zSpinBox">
                <property name="singleStep">
                 <double>0.010000000000000</double>
                </property>
               </widget>
              </item>
              <item>
               <spacer name="horizontalSpacer_2">
                <property name="orientation">
                 <enum>Qt::Horizontal</enum>
                </property>
                <property name="sizeHint" stdset="0">
                 <size>
                  <width>40</width>
                  <height>20</height>
                 </size>
                </property>
               </spacer>
              </item>
             </layout>
            </item>
           </layout>
          </widget>
         </item>
         <item row="2" column="0">
          <spacer name="verticalSpacer">
           <property name="orientation">
            <enum>Qt::Vertical</enum>
           </property>
           <property name="sizeHint" stdset="0">
            <size>
             <width>20</width>
             <height>40</height>
            </size>
           </property>
          </spacer>
         </item>
        </layout>
       </item>
      </layout>
     </widget>
     <widget class="QGroupBox" name="groupBox_2">
      <property name="title">
       <string>Preview</string>
      </property>
      <layout class="QGridLayout" name="gridLayout_2">
       <item row="0" column="0">
        <layout class="QVBoxLayout" name="previewLayout"/>
       </item>
      </layout>
     </widget>
    </widget>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections/>
</ui>
